packages/algob/src/types.ts

```diff
@@ -370,6 +370,7 @@ export interface ASADeploymentFlags extends TxParams {
 export interface ASCDeploymentFlags {
   funder: Account
   fundingMicroAlgo: number
+  mode: string
```

Comment:
This should be enum, not a string.

Please explain in Discord why you need this and what values you want to add.